homeassistant/components/tado/water_heater.py

```diff
@@ -146,19 +149,18 @@ def __init__(
         self._min_temperature = min_temp
         self._max_temperature = max_temp
```

Comment:
```suggestion
        self._min_temperature = min_temp or TADO_DEFAULT_MIN_TEMP
        self._max_temperature = max_temp or TADO_DEFAULT_MAX_TEMP
```

So we can avoid the check on the property